export/exportconfig.go

```diff
@@ -3,21 +3,23 @@ package export
 import (
 	cc "github.com/pivotalservices/cf-mgmt/cloudcontroller"
 	"github.com/pivotalservices/cf-mgmt/config"
-	"github.com/pivotalservices/cf-mgmt/organization"
-	"github.com/pivotalservices/cf-mgmt/space"
+	kOrg "github.com/pivotalservices/cf-mgmt/organization/constants"
```

Comment:
This is pretty unidiomatic - most of the time in Go we put constants in the package that they relate to.  Additionally, if you have to alias a package import (most of the time you shouldn't, this is usually a code smell) the convention is usually to use all lowercase.